screens/PrepareFace.tsx

```diff
@@ -1,13 +1,138 @@
+} from "react-native";
 import { Button, Text } from "../components/Themed";
-import { Ionicons } from "@expo/vector-icons";
-const PrepareFace: FunctionComponent = () => {
+import LetsStartRecording from "./LetsStartRecording";
+
+const SpacingDiv = () => {
+  return <View style={{ flex: 1 }} />;
+};
+
+interface BottomInfoPromptProps {
+  inScreen: boolean;
+}
+
+const BottomInfoPrompt: FunctionComponent<
+  BottomInfoPromptProps & PrepareFaceProps
+> = ({ inScreen, finished }): ReactElement => {
   const navigation = useNavigation();
+  return (
+    <View style={BottomInfoPromptStyles.container}>
+      {!inScreen && (
+        <View style={BottomInfoPromptStyles.adjustCameraView}>
+          <Text style={BottomInfoPromptStyles.adjustCameraText}>
+            please adjust your camera
+          </Text>
+        </View>
+      )}
+      <View style={BottomInfoPromptStyles.bottomInfoContainer}>
+        <SpacingDiv />
+        <Text style={BottomInfoPromptStyles.text}>
+          Please check that your face is visible in the window below. When your
+          face is centered, please press the button.
+        </Text>
+        <SpacingDiv />
+        <Button
+          style={{
+            ...BottomInfoPromptStyles.button,
+            backgroundColor: inScreen ? "#0038FF" : "#8D8D8D",
+          }}
+          disabled={!inScreen}
+          onPress={finished}
+        >
+          <Text style={BottomInfoPromptStyles.buttonText}>
+            My face is in the window
+          </Text>
+        </Button>
+        <SpacingDiv />
+      </View>
+    </View>
+  );
+};
+
+const BottomInfoPromptStyles = StyleSheet.create({
+  container: {
+    position: "absolute",
+    bottom: 0,
+    left: 0,
+    width: "100%",
+    minHeight: "25%",
+  },
+  bottomInfoContainer: {
+    backgroundColor: "rgba(0, 0, 0, 0.67)",
+    justifyContent: "space-around",
+    paddingHorizontal: "5%",
+    paddingBottom: 20,
+  },
+  text: {
+    flex: 3,
+    fontWeight: "400",
+    fontSize: 18,
+    color: "white",
+    padding: 10,
+    textAlign: "center",
+  },
+  button: {
+    flex: 3,
+    padding: 10,
+  },
+  buttonText: {
+    flex: 1,
+    textAlign: "center",
+    fontSize: 20,
+    fontWeight: "400",
+    color: "white",
+  },
+  adjustCameraView: {
+    borderRadius: 24,
+    backgroundColor: "black",
+    width: "75%",
+    marginHorizontal: "12.5%",
+    marginBottom: 5,
+  },
+  adjustCameraText: {
+    textAlign: "center",
+    padding: 5,
+    color: "white",
+    flex: 1,
+  },
+});
+
+interface Face {
+  bounds: {
+    origin: { x: number; y: number };
+    size: { width: number; height: number };
+  };
+}
+
+interface PrepareFaceProps {
+  finished: () => void;
+}
+
+const PrepareFace: FunctionComponent<PrepareFaceProps> = ({ finished }) => {
   const [seeingFace, setSeeingFace] = useState(true);
   const [permissions, setHasPermission] = useState(false);
+  let dimensions = useWindowDimensions();
+  dimensions = Dimensions.get("screen");
```

Comment:
why not just `let dimensions = Dimensions.get('screen');`?

Reply:
`useWindowDimensions` is a hook, so when the size changes (like iPad changing from landscape to portrait), then the view gets re-rendered. Dimensions does not do that for us.